FHIR/GoFSH
# Fix types root in package.json

**Description:** This sets the types root to `dist/index.d.ts`. Previously, the types root pointed to `dist/app.d.ts`, which prevented GoFSH from being imported property in other applications.

**Testing Instructions:**
Repro the problem:
1. Download the attached simple TS project that imports GoFSH: [ts-project.zip](https://github.com/user-attachments/files/17561559/ts-project.zip)
2. Unzip it and run `npm install`
3. Run `npm run build` and observe compile errors

Test the solution:
1. Check out the branch from this PR
2. Run `npm pack` to create the module file `gofsh-2.3.1.tgz`
3. Copy `gofsh-2.3.1.tgz` to the simple TS project you downloaded
4. In that simple TS project, run `npm install --save file:gofsh-2.3.1.tgz`
5. Run `npm run build` and observe that it builds cleanly

**Related Issue:** Discovered and discussed in [FSH VS Code PR #97](https://github.com/FHIR/vscode-fsh/pull/97#discussion_r1821094864)
